tests/components/prometheus/test_init.py

```diff
@@ -162,16 +188,22 @@ async def test_view_empty_namespace(
         "Objects collected during gc" in body
     )
 
-    assert (
-        'entity_available{domain="sensor",'
-        'entity="sensor.radio_energy",'
-        'friendly_name="Radio Energy"} 1.0' in body
+    MetricsTestHelper._perform_sensor_metric_assert(
+        "entity_available",
+        "1.0",
+        "Radio Energy",
+        "radio_energy",
+        body,
+        device_class=SensorDeviceClass.POWER,
```

Comment:
Could you please use strings for checking label values instead of referring to constants in other modules?